src/platform/silabs/platformAbstraction/GsdkSpam.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include <em_device.h>
 #include <lib/support/CodeUtils.h>
+#include <platform/DiagnosticDataProvider.h>
+#include <platform/silabs/SilabsConfig.h>
```

Comment:
why do we need those added include in this cpp? There is no other code added to this file

Reply:
That appears to have been left behind as I moved the logic to SilabsPlatform.cpp